app/frontend/app/controllers/pro.js

```diff
@@ -104,7 +103,9 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
         key: PreloadStore.get('stripe_key'),
         name: "Hummingbird PRO",
         description: this.get('selectedPlan.name'),
-        token: tokenCallback
+        token: tokenCallback,
+        amount: parseInt(this.get('selectedPlan.amount') * 100),
```

Comment:
parseInt without a radix parameter of 10 isn't safe.  Specify it just to be safe.

Also shouldn't the multiplication be after it's turned into an integer, not before?